docs/process.rst

```diff
@@ -147,6 +147,9 @@ Making Releases
 Checklist for Making a Stable Release
 -------------------------------------
 
+The release should be built in Vivid or later. Earlier versions (Trusty,
+Utopic) will run into problems with running the functional tests.
```

Comment:
I added more detail in the "You need Vivid" note just before the functional tests.